submissions/ik-web/html-movie-seat-booking/index.html

```diff
@@ -0,0 +1,280 @@
+<!Doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Movie Seat Booking</title>
+    <link rel="icon" href="img/favicon.ico" type="image/x-icon">
+    <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+    <main class="main">
+        <section class="movie-info">
+            <h1 class="movie-info__title">Monster Hunter</h1>
+            <div class="movie-info__inner">
+                <div class="movie-info__poster">
+                    <img src="img/movie.jpg" alt="movie poster">
+                </div>
+
+                <ul class="movie-info__about">
+                    <li class="movie-info__about-item about-item__format">3D</li>
+                    <li class="movie-info__about-item about-item__map">Kyiv, SEC "Lavina", Hall 13</li>
+                    <li class="movie-info__about-item about-item__date">2021.02.04 Thursday</li>
+                    <li class="movie-info__about-item about-item__time">Time 18:50 - 21:39</li>
+                </ul>
+            </div>
+        </section>
+
+        <section class="booking-seats">
+            <h2 class="booking-seats__title">Choose your seats</h2>
+
+            <div class="booking-seats__cinema-hall">
+                <div class="cinema-hall__screen">screen</div>
+                <form class="cinema-hall__seats" id="seats-form">
+
+                    <ol class="cinema-hall__row">
+                        <li>
+                            <label>
+                                <input type="checkbox" class="cinema-hall__check" value="1">
```

Comment:
Add styles on `:hover` and `:focus` states.

Reply:
Added. Thank you for this comment!